### DISCUSSION TITLE
Table 10, Panel 1

### DISCUSSION THREAD

**Discussion by manning-ncsa** [cssi-pi-community/2022-meeting#13] [Panel 1 roundtable (Monday morning)]
Table 10, Panel 1 discussion

**Comment by manning-ncsa** [comment]
Marisa Brazil's talk was an excellent summary of the important career issues in this field.

**Comment by manning-ncsa** [comment]
The inadequancy of the traditional academic structures to support the career growth and recognition was accurately identified by Madicken Munk.

**Comment by manning-ncsa** [comment]
Academic professors are expected to devote some fraction of time to reviewing papers and participating in review panels. Research software engineers could likewise be expected to contribute to upstream software projects and pull request reviews, where that activity is rewarded and factored into time allocations.

**Comment by volkerblum** [comment]
One aspect of our discussion focused on simple obstacles faced by research software scientists inside academic structures, including job titles that do not enjoy sufficient recognition by the wider academic community (e.g. review panels). One example is "specialist" instead of "scientist" in a job title (there are others.).

Another topic that was briefly touched upon is the demographic background of some those pursuing science careers, where there may be an imbalance already stemming from family/social background (e.g., children from already academically focused families).
